tests/unit_tests/test_drone.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from SDA import WaypointHolder
+from SUASSystem import *
+import unittest
+
+class DroneTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.drone = Drone(np.array([0,0,0]), np.array([[ 7, 10, 13], [21, 24, 27]]))
+
+    def test_set_drone_pos(self):
+        self.drone.set_drone_position(np.array([1,1,1]))
+        self.assertEquals(np.array([1,1,1]).any(), self.drone.point.any())
+
+    def test_add_waypoint(self):
+        self.drone.add_waypoint(np.array([20,20,20]))
```

Comment:
You should remove all waypoints before you call this. The tests do not always execute from top of the file to the bottom, so you should clear the waypoints to ensure that the test completes properly.

Reply:
done